CHANGELOG.rst

```diff
@@ -3,17 +3,22 @@
 0.60.0
 ------
 
+Renku 0.60.0 squashes an issue that has been tripping up many users when connecting their GitHub
+account, as well as various UX polish and bugfixes.
```

Comment:
it's odd having `polish` be singular, but then plural of polish also sounds weird